app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,28 +1,46 @@
-import type { ChangeEvent } from 'react';
-import { PasswordToggle } from '@18f/identity-password-toggle';
+import { PageHeading, Accordion } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
 import { FormStepsButton } from '@18f/identity-form-steps';
+import PersonalInfoSummary from './personal-info-summary';
+import { PasswordToggle } from '@18f/identity-password-toggle';
 import { Alert } from '@18f/identity-components';
-import type { FormStepComponentProps } from '@18f/identity-form-steps';
 import StartOverOrCancel from '../../start-over-or-cancel';
-import type { VerifyFlowValues } from '../../verify-flow';
+import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import type { ChangeEvent } from 'react';
+import type { VerifyFlowValues } from '../..';
 
-interface PasswordConfirmStepStepProps extends FormStepComponentProps<VerifyFlowValues> {}
+interface PasswordConfirmStepProps extends FormStepComponentProps<VerifyFlowValues> {}
 
-function PasswordConfirmStep({ errors, registerField, onChange }: PasswordConfirmStepStepProps) {
+function PasswordConfirmStep({ errors, registerField, onChange, value }: PasswordConfirmStepProps) {
   return (
     <>
       {errors.map(({ error }) => (
         <Alert key={error.message} type="error" className="margin-bottom-4">
           {error.message}
         </Alert>
       ))}
-      <PasswordToggle
-        ref={registerField('password')}
-        type="password"
-        onInput={(event: ChangeEvent<HTMLInputElement>) => {
-          onChange({ password: event.target.value });
-        }}
-      />
+      <PageHeading>{t('idv.titles.session.review', { app_name: 'Login.gov' })}</PageHeading>
```

Comment:
As of #6342, we now expose `appName` as a common config value, so we don't have to hard-code it (to avoid another #5225 + #5498).

```ts
import { getConfigValue } from '@18f/identity-config';

// ...

return <PageHeading>{t('idv.titles.session.review', { app_name: getConfigValue('appName') })}</PageHeading>;
```

Reply:
why return <PageHeading> ...  why not just <PageHeading> ...

Reply:
Only meant it as an example pseudo-code. No return necessary where it's used.